packages/animated-qr/src/AnimatedQRScanner/BaseQRScanner.tsx

```diff
@@ -32,36 +36,48 @@ export const BaseQRScanner = ({
   }, []);
 
   useEffect(() => {
-    const videoElement = document.getElementById(VIDEO_ID) as HTMLVideoElement;
+    const videoElement = document.getElementById(
+      VIDEO_ID
+    ) as HTMLVideoElementScanPreview;
```

Comment:
A quick trick for extending the original object with a custom property, and maintaining types